cdap-explore/src/main/java/co/cask/cdap/hive/stream/HiveStreamInputFormat.java

```diff
@@ -194,7 +183,7 @@ public InputSplit createSplit(Path path, Path indexPath, long startTime, long en
 
     private final String opClassName;
 
-    private CompareOp(String opClassName) {
+    CompareOp(String opClassName) {
```

Comment:
IntelliJ have a warning for having private, since enum constructor is always private, hence modifier is unnecessary.